clang/lib/3C/CheckedRegions.cpp

```diff
@@ -43,18 +43,21 @@ bool CheckedRegionAdder::VisitCompoundStmt(CompoundStmt *S) {
   llvm::FoldingSetNodeID Id;
   ast_type_traits::DynTypedNode DTN = ast_type_traits::DynTypedNode::create(*S);
 
+  auto &PState = Info.getPerfStats();
   S->Profile(Id, *Context, true);
   switch (Map[Id]) {
   case IS_UNCHECKED:
     if (isParentChecked(DTN) && getFunctionDeclOfBody(Context, S) == nullptr) {
       auto Loc = S->getBeginLoc();
       Writer.InsertTextBefore(Loc, "_Unchecked ");
+      PState.incrementNumUnCheckedRegions();
```

Comment:
This is adding checked regions that we are inferring, is that right? What about checked regions already present in the code?

Reply:
We only compute those that are added by our tool.